docs/sdk/v3/guides/01-background.md

```diff
@@ -0,0 +1,64 @@
+
+If you are connecting to a wallet extension, these wallets embed a provider directly into the Javascript window object as `window.ethereum` which can also be used as input into `ethers.js` to construct a provider as well as to access the user's wallet information and complete transactions through the wallet.
+
+## Uniswap's Runnable Examples
+
+Each guide is accompanied and driven by [runnable examples](https://github.com/Uniswap/examples/tree/main/v3-sdk) using React to provide a basic UI for interacting with the example. Each examples provides relevant options to run against such as running against a local blockchain, connecting to the Ethereum mainnet directly, or using a wallet extension.
+
+Inputs and environment settings are configured in each example's `config.ts` and allows for simple setup and configuration.
+
+### Developing and Testing
+
+To test your code, we recommend utilizing a local fork of the Ethereum mainnet. To help facilitate easy testing, each example includes a quickstart for running this local chain with a test wallet. To further test, we also recommend using a wallet extension and connecting to this local chain. Finally, each example can be run against the Ethereum mainnet if desired. Full development instructs can be found in the `README.md` of each code example.
+
+### Utility Libraries
+
+To allow the guides to focus on the SDK's core functionality, basic building blocks can be found in each example's `libs` folder. The exported functionality from these files is intended to be the minimum needed for each example and not a complete library for production usage. These also include storing core constants such as definitions for tokens, ABI's, and blockchain addresses that can distract from the core concepts. Below are summaries of the helping libraries you will encounter.
+
+#### Provider Utilities
+
+`provider.ts` wraps the basics of `ethers.js` and connecting to wallet extensions into an abstracted view of a provider, a wallet address, and the ability to send transactions. It also helps abstract the configured environment you wish to run against in your example without making code changes outside of your configuration.
+
+#### Wallet Utilities
+
+`wallet.ts` offers the ability to query a wallet (whether connected via an extension or defined in code/config) for its balances and other essential information.
+
+#### Display Utilities
+
+`conversion.ts` provides display and light math wrappers to help show human readable prices when dealing with currency amounts (typically stored as raw numbers and the decimal placement separate for precision reasons) in the form of two functions: `fromReadableAmount` and `toReadableAmount`
+
+## Notable SDK Structures and Concepts
+
+When working with the SDK it can be helpful to understand some of the design choices and why they are needed. Below you can find a few important concepts.
+
+### ABI's
+
```

Comment:
The SDK does not really provide the definitions - that's why we use a bunch of different NPM packages. We should call that out!

Reply:
> A **reading time** feature/call-out would also be great to highlight how light these guides are

Thoughts on doing this at the end across all the docs for standardization?